### DISCUSSION TITLE
Why I can't select my patched font?

### DISCUSSION THREAD

**Discussion by augustocdias** [kovidgoyal/kitty#5621] [Q&A] [answered]
I'm on macOS and I have patched my font to add more symbols but I can't manage to make kitty read it... 

The output of `kitty +list-fonts | grep Fira` is:

```
Fira Code
    Fira Code Bold
    Fira Code Light
    Fira Code Medium
    Fira Code Regular
    Fira Code Retina
    Fira Code SemiBold
Fira Code iCursive S12
    Fira Code iCursive S12 Regular
FiraCode Nerd Font Mono
    FiraCode Nerd Font Mono Bold
    FiraCode Nerd Font Mono Light
    FiraCode Nerd Font Mono Medium
    FiraCode Nerd Font Mono Regular
    FiraCode Nerd Font Mono Retina
    FiraCode Nerd Font Mono SemiBold
```

Here's a print of my installed FiraCode variants. They're located in `/Users/augusto/Library/Fonts`
![image](https://user-images.githubusercontent.com/1080789/197728674-95ff4547-ff4e-447d-b496-ffb236a4f0a4.png)

How can I load those `Complete` fonts and kitty seems to load only the Mono variants of the Nerd Font's ones. How can I load the non Mono ones? I prefer the non Mono...

**Comment by kovidgoyal** [comment, marked as answer]
https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font

**Reply by glensc** [reply]
I've done the exact steps:
1. Download https://github.com/ryanoasis/nerd-fonts/releases/download/v2.3.2/NerdFontsSymbolsOnly.zip
2. Unpack it to ~/Library/Fonts
3. Edit config, add to config `symbol_map U+23FB-U+23FE,U+2665,U+26A1,U+2B58,U+E000-U+E00A,U+E0A0-U+E0A3,U+E0B0-U+E0C8,U+E0CA,U+E0CC-U+E0D2,U+E0D4,U+E200-U+E2A9,U+E300-U+E3E3,U+E5FA-U+E634,U+E700-U+E7C5,U+EA60-U+EBEB,U+F000-U+F2E0,U+F300-U+F32F,U+F400-U+F4A9,U+F500-U+F8FF Symbols Nerd Font Mono`
4. Run `kitty --debug-font-fallback`
5. Test `print '\u23FB'` -> should show: ⏻ (as seen in this browser)

but the font doesn't appear in list fonts and test print also fails. what I'm missing?

```
~ ➜ kitty --debug-font-fallback
[053 14:17:07.482785] The font Symbols Nerd Font Mono was not found, falling back to Menlo
[053 14:17:07.482870] Preloaded font faces:
[053 14:17:07.482891] normal face: /System/Library/Fonts/Menlo.ttc
[053 14:17:07.482912] bold face: /System/Library/Fonts/Menlo.ttc
[053 14:17:07.482929] italic face: /System/Library/Fonts/Menlo.ttc
[053 14:17:07.482947] bi face: /System/Library/Fonts/Menlo.ttc
[053 14:17:07.482975] Symbol map faces:
[053 14:17:07.482992] /System/Library/Fonts/Menlo.ttc

~ ➜ kitty +list-fonts
Andale Mono
    Andale Mono Regular

Courier New
    Courier New Bold
    Courier New Bold Italic
    Courier New Italic
    Courier New Regular

Menlo
    Menlo Bold
    Menlo Bold Italic
    Menlo Italic
    Menlo Regular

Monaco
    Monaco Regular

PT Mono
    PT Mono Bold
    PT Mono Regular

~ ➜ ls ~/Library/Fonts 
Inconsolata[wdth,wght].ttf
Symbols-1000-em Nerd Font Complete Mono Windows Compatible.ttf
Symbols-1000-em Nerd Font Complete Mono.ttf
Symbols-1000-em Nerd Font Complete Windows Compatible.ttf
Symbols-1000-em Nerd Font Complete.ttf
Symbols-2048-em Nerd Font Complete Mono Windows Compatible.ttf
Symbols-2048-em Nerd Font Complete Mono.ttf
Symbols-2048-em Nerd Font Complete Windows Compatible.ttf
Symbols-2048-em Nerd Font Complete.ttf
~ ➜ 
```

**Reply by glensc** [reply]
Nothing worked. including installing with Font Book.

Various posts in the internet just said the same:
- https://www.geekbits.io/how-to-install-nerd-fonts-on-mac/
- https://apple.stackexchange.com/questions/94842/how-to-use-font-installed-without-rebooting-the-system

so, ended up logging out and after that fonts appear in list fonts including ones installed with Font Book and ones installed with brew. 

Is some macOS 13 update at fault (I'm at 13.1)?

Should the log out instruction be added to the documentation?
